web/themes/custom/pineapple/sass/components/_hero-image.scss

```diff
@@ -48,6 +48,19 @@
         margin-top: $unit * 3;
         display: inline-block;
 
+        &:hover {
+          background-color: transparentize($white, 0.2);
+          color: $primary;
+
+          a {
+            color: $primary;
+          }
+        }
```

Comment:
add a transition on this, the instant change is too jarring

Reply:
okay done